Assignment3/calculatingWithFunctions.js

```diff
@@ -0,0 +1,113 @@
+function zero() {
+
+  if(arguments.length>0){// if there is argument then we simply pass value to the function which is in argument
+    return arguments[0](0)
+  }else{ // we will return integer
+    return 0
+  }
+
+}
+function one() {
+
+  if(arguments.length>0){
+    return arguments[0](1)
+  }else{
+    return 1
+  }
+
+}
+function two() {
+
+  if(arguments.length>0){
+    return arguments[0](2)
+  }else{
+    return 2
+  }
+
+}
+function three() {
+
+  if(arguments.length>0){
+    return arguments[0](3)
+  }else{
+    return 3
+  }
+
+}
+function four() {
+
+  if(arguments.length>0){
+    return arguments[0](4)
+  }else{
+    return 4
+  }
+
+}
+
+function five() {
+
+  if(arguments.length>0){
+    return arguments[0](5)
+  }else{
+    return 5
+  }
+
+}
+function six() {
+
+  if(arguments.length>0){
+    return arguments[0](6)
+  }else{
+    return 6
+  }
+
+}
+function seven() {
+

```

Comment:
Functions one to Seven can be combined as one function. pls re-work

Reply:
We go the functions from KATA compiler , there functions were divided, we just had to add our code inside function. That's why we did like this